src/parser.coffee

```diff
@@ -400,6 +451,7 @@ class Parser
 
         when "<"
           # < Label
+          cmdtype = 'label'
```

Comment:
Should this go on the open label `>`?

Reply:
It's either or since they're a matched set, so long as it's not in both.